quick-start/src/main/ui/app/components/flows-new/models/flow.model.ts

```diff
@@ -27,7 +27,7 @@ export class Flow {
   constructor() {}
 
   get status(): string {
-    return this.latestJob['status'];
+    return (this.latestJob['status']) ? this.latestJob['status'] : 'Never Run';
```

Comment:
just to ensure you don't hit an issue with an empty latestJob, change this to:

`return (this.latestJob && this.latestJob['status']) ? this.latestJob['status'] : 'Never Run';`